### PR TITLE
Public Preview of Mixed Reality Cloud Service

### PR DESCRIPTION
**Pull request by XiangyuL-Microsoft**
This is the first code check-in of Mixed Reality Azure Resource Management .NET SDK.

All checked-in code (only except the generate.ps1) had been reviewed and checked-in the private Azure GitHub repo:

https://github.com/Azure/azure-sdk-for-net-pr/tree/NetSdkMixedReality

### CODE REVIEW

File: src/SDKs/MixedReality/Management.MixedReality/Management.MixedReality.csproj [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  
+  1. List all available operations;
+  2. Create/Delete specified Spatial Anchors Account;
+  3. Add/Modify/Remove tags of specified Spatial Anchors Account;
+  4. List all Spatial Anchors Account(s) in certain Resource Group or Subscription.
+]]>
+    </PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup>
```
Comment (shahabhijeet) [inline src/SDKs/MixedReality/Management.MixedReality/Management.MixedReality.csproj]: @XiangyuL-Microsoft take a look at other projects and model your project file accordingly.
You cannot specify target fx in your project file, plus now all .NET SDKs support more target fx than just these two.
Look at https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj
and model your project file accordingly.

File: src/SDKs/MixedReality/MixedReality.Tests/MixedReality.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```
Comment (shahabhijeet) [inline src/SDKs/MixedReality/MixedReality.Tests/MixedReality.Tests.csproj]: @XiangyuL-Microsoft model your test project looking at other test projects in the repository.
Take a look at this
https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Use generate.ps1 to generate your SDK.
Pick any generate.ps1 and model your generate.ps1 accordingly.
It is important you use generate.ps1, because that creates the required meta data that we need for the SDK.

File: src/SDKs/MixedReality/MixedReality.Tests/MixedReality.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>MixedReality.Tests</PackageId>
+    <Description>MixedReality.Tests Class Library</Description>
+    <AssemblyName>MixedReality.Tests</AssemblyName>
+    <VersionPrefix>0.9.0-preview</VersionPrefix>    
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
```
Comment (shahabhijeet) [inline src/SDKs/MixedReality/MixedReality.Tests/MixedReality.Tests.csproj]: @XiangyuL-Microsoft please remove the reference to Resource Manager
This reference is implicitly added to every test project.
Adding this separately will hamper our ability to update common references quickly

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
everything else looks good. Please make the update about reference and this PR is ready to be merged.

**Comment by XiangyuL-Microsoft** [comment]
> everything else looks good. Please make the update about reference and this PR is ready to be merged.

Not intentionally refer the Microsoft.Azure.ResourceManager NuGet Package. but just follow Compute.Tests:

https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj

File: src/SDKs/MixedReality/MixedReality.Tests/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM components.  If you need to access a type in this assembly from 
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("1050e361-5461-40c6-a866-6b90e01aac9c")]
+[assembly: AssemblyVersion("1.0.0.0")]
```
Comment (shahabhijeet) [inline src/SDKs/MixedReality/MixedReality.Tests/Properties/AssemblyInfo.cs]: @XiangyuL-Microsoft I just noticed, you assembly verison and assembly File version do not match to nuget version.
Make it 0.9.0.0 for each of those versions.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**